scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -1,3 +1,11 @@
 &.page-civicrm-a {
   @include civihr_table();
+
+  table tr.forked {
+      font-weight: normal;
+  }
+
+  #crm-main-content-wrapper .crm-submit-buttons {
+    box-shadow: none;
```

Comment:
isn't this supposed to be something applied to *any* button in the core screens? why do you have to specify it for this particular button?